tests/Browser/Pages/Backend/Borrows/AdminListBorrowsTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace Tests\Browser\Pages\Backend\Borrows;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use App\Model\User;
+use App\Model\Category;
+use App\Model\Borrow;
+use App\Model\Book;
+use Illuminate\Support\Facades\DB;
+use Faker\Factory as Faker;
+
+class AdminListBorrowsTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+
+    const NUMBER_RECORD_CREATE = 18;
+
+    /**
+     * Override function setUp()
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+       parent::setUp();
+
+       $this->createAdminUser();
+    }
+
+    /**
+     * A Dusk test show view list borrows.
+     *
+     * @return void
+     */
+    public function testListBorrows()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::first())
+                    ->visit('/admin')
+                    ->clickLink('Borrows')
+                    ->assertPathIs('/admin/borrows')
+                    ->assertSee('List Borrower');
+        });
+    }
+
+    /**
+     * A Dusk test list borrows if empty data borrows
+     *
+     * @return void
+     */
+    public function testListBorrowsEmpty()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::first())
+                    ->visit('/admin/borrows')
+                    ->assertSee('List Borrower');
+            $elements = $browser->elements('#list-borrows tbody tr');
+            $this->assertCount(0, $elements);
+            $this->assertNull($browser->element('.paginate'));
+        });
+    }
+
+    /**
+    * A Dusk test show record with table borrows has data.
+    *
+    * @return void
+    */
+    public function testShowRecord()
+    {
+        $this->makeData(4);
```

Comment:
đã config rồi thì dùng luôn em sao lại phải để số 4 vào làm gì??
$this->makeData(self::NUMBER_RECORD_CREATE);